todo_app/data/todo_items.py

```diff
@@ -123,9 +124,11 @@ def delete_todo_card(card_id):
 
     for c in collections:
         collection = db[c]
-        result = collection.delete_one({'_id' : card_id})
-        print(result)
-        break
+        for card in collection.find({}): 
+            if str(card['_id']) == str(card_id):
+                result = collection.delete_one({'_id' : card['_id']})
+                print(result)
+                break
```

Comment:
I know we talked about this briefly; this works and I'm happy with that but worth noting that the relevant syntax to avoid the inner loop is:
```suggestion
        collection.delete_id({'_id': ObjectId(card_id)})
```